meta-mender-toradex-nxp/recipes-bsp/u-boot/patches/0005-fw_env_main.c-Fix-incorrect-size-for-malloc-ed-strin.patch

```diff
@@ -0,0 +1,28 @@
+From 17236152bf80436567bf3f1f27af3915364ec0b6 Mon Sep 17 00:00:00 2001
+From: Kristian Amlie <kristian.amlie@northern.tech>
+Date: Wed, 4 Apr 2018 10:01:04 +0200
+Subject: [PATCH 5/5] fw_env_main.c: Fix incorrect size for malloc'ed string.
+
+Using sizeof gives the size of the pointer only, not the string.
+
+Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>
```

Comment:
I know we have not really enforced this, but what do you think about adding `Upstream-status:` tags on these patches.

Based on https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines, Section "Patch Header Recommendations: Upstream-Status".

Reply:
Sure, I can add it for convenience. Our patch flow is too low that I would bother about enforcing this now though.